### DISCUSSION TITLE
Is the zombie mode ram bandwith limited?

### DISCUSSION THREAD

**Discussion by Auserone** [Lolliedieb/lolMiner-releases#245] [General]
The question is - faster memory sticks means better zombie mode performance? Not complaining, just thinking about. 

My results:

Single RX570 in x16 mode 28.3 mh/s
Dual RX570 in x8 mode 27.3 mh/s each, with a single DDR4 2133 ram stick.^^

And thank you of course - x16 zombie mode is a brilliant idea.

_^^correction, this is a crucial ddr4 2400 ram stick._

**Comment by thowell76** [comment]
Are those speeds with ETH?   I cannot get my RX570 to mine over 10 mh/s on ETH.   I have only used auto for zombie mode, when I try to use it, it crashes.   So I am curious if their are any other custom settings to get that hashrate.

**Reply by Auserone** [reply]
No custom settings, it just works.

Maybe the following might help:

1. Lower the card's mem speed.
2. Check BIOS for pcie speeds.
3. Check the health of the RAM stick(s) with MemTest86
4. Test the card in another pc if the above does not helps, or test another card in the same pc.
5. And it only works on linux, so do not use windows for zombie mode.

**Reply by bullet97** [reply]
Guys, anybody try zombie mode with old matherboard with ddr2? what is the result?

**Comment by Auserone** [comment]
Some new results:
Epoch 394 Dual RX570 in x8 mode 27.3 mh/s each, with a single DDR4 2400 ram stick.
Epoch 395 Dual RX570 in x8 mode 26.6 mh/s each, with a single DDR4 2400 ram stick.

Epoch 395 Dual RX570 in x8 mode 27.9 mh/s each, with dual DDR4 2400 ram stick.

Epoch 395 Single RX570 in x16 mode went down from 28.3+ to 28.1-28.2 with a single DDR4 2400 ram stick.

So it seems RAM speed limited for now in the used setup.

**Comment by Auserone** [comment]
lolminer 1.24a vs. 1.22 dual x8 setup.

1.22 Epoch 395 Dual RX570 in x8 mode 26.6 mh/s each, with a single DDR4 2400 ram stick vs.
1.24a Epoch 395 Dual RX570 in x8 mode 25.9 mh/s each, with a single DDR4 2400 ram stick

**Comment by Auserone** [comment]
Had the chance to measure an RX470 mining version as single x16 card, it is doing 26.1 with single DDR4, epoch 396, while RX570 is still going strong around 28.1 mh/s.

**Reply by Migel-del** [reply]
Hi. Can you contact me in telegram? @titomirtm
